src/tests/frontend/index.html

```diff
@@ -16,6 +16,7 @@
 
     <script src="../../static/js/require-kernel.js"></script>
     <script src="../../static/js/vendors/jquery.js"></script>
+    <script src="lib/sendkeys.js"></script>
```

Comment:
Isn't this handled by the `includeSendkeys` parameter?

Reply:
That's why I wanted to get rid of the 3 additional instances of jquery. We have separate jq instances on the test main site (index.html that starts mocha), in pad.html, ace2inner and enforced by (obsolete helper.init or) the logic in this PR that ensures it's also available as a global in ace_outer iframe.

It's kind of confusing which instance is used. When running the admin tests we don't need sendkeys because all text fields are manipulated with `val()` and jquery is already included via script elements (however, testing if this is the case wouldn't hurt now that getFrameJquery also checks if it's already defined).

I can remove this, as it won't make a difference (sendkeys isn't used in the context of the "main site" and prefetching the file won't have any performance impact). I think getting rid of three additional jquery instantions every time helper.newPad is called would make some small performance improvement, but it's something for another PR.

Btw: In the long term I'd like to get rid of jquery in core entirely, because it's very large compared to our code. But as this means getting rid of farbtastic and other plugins (most of them seem to be very outdated) and some plugins require jquery to be in core, this is some kind of bigger task.

Reply:
btw this was needed. `includeSendkeys` is not considered in the index.html context (where helper.js is included)